docs/ABI/Mangling.rst

```diff
@@ -638,7 +638,7 @@ Types
   type ::= protocol-list 'p'                 // existential type
   type ::= protocol-list superclass 'Xc'     // existential type with superclass
   type ::= protocol-list 'Xl'                // existential type with AnyObject
-  type ::= protocol-list 'y' (type* '_')* type* retroactive-conformance* 'XP'   // parameterized protocol type
+  type ::= protocol-list 'y' (requirement* '_')* requirement* 'XP'   // constrained existential type
```

Comment:
Why are there mulltiple levels of requirements here?

Reply:
Thinko, corrected.